examples/stubbing-spying__intercept/cypress/integration/glob-matching-intercept-url.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('intercept', () => {
+  cy.intercept('/api/roles')
+  // will match https://example.com/api/roles
+  // will not match https://example.com/api/roles?country=US
+
+  cy.intercept('/api/roles*')
+  // will match https://example.com/api/roles
+  // will match https://example.com/api/roles?country=US
+
+  cy.intercept('/api/roles?*utm_source=+(MyApp|https*)')
+  // will match https://example.com/api/roles?utm_medium=Nav&utm_campaign=Docs&utm_source=MyApp
```

Comment:
Could you remove either the `utm_medium` or `utm_campaign` parameter from the examples just to make the lines a little shorter and easier to read?

Reply:
Done